azure/packages/azure-client/package.json

```diff
@@ -75,7 +76,6 @@
     "eslint": "~8.6.0",
     "mocha": "^10.0.0",
     "rimraf": "^2.6.2",
-    "start-server-and-test": "^1.11.7",
```

Comment:
It seems like devDependencies are not installed in CI: https://dev.azure.com/fluidframework/internal/_build/results?buildId=79008&view=logs&j=5f8fe092-e504-55d5-bea6-4c19fb56d81e&t=bca09894-4e59-530e-6279-885c563a2ffe&l=9. @tylerbutler let me know if you have more context here. I don't see pipelines running in --production mode.

Reply:
start-server-and-test is now passed as extraDependency during FRS tests with azure local service.